Content.IntegrationTests/Tests/Pulling/PullTest.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Threading.Tasks;
+using Content.Server.GameObjects.Components.Pulling;
+using Content.Shared.GameObjects.Components.Pulling;
+using Content.Shared.Physics.Pull;
+using NUnit.Framework;
+using Robust.Shared.GameObjects;
+using Robust.Shared.GameObjects.Components;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Interfaces.Map;
+using Robust.Shared.IoC;
+using Robust.Shared.Map;
+
+namespace Content.IntegrationTests.Tests.Pulling
+{
+    [TestFixture]
+    [TestOf(typeof(SharedPullableComponent))]
+    [TestOf(typeof(SharedPullerComponent))]
+    [TestOf(typeof(PullController))]
+    public class PullTest : ContentIntegrationTest
+    {
+        [Test]
+        public async Task AnchoredNoPullTest()
+        {
+            var server = StartServerDummyTicker();
+
+            await server.WaitAssertion(() =>
+            {
+                var mapManager = IoCManager.Resolve<IMapManager>();
```

Comment:
Should this be server.ResolveDependency or does it not matter if there is no client?

Reply:
I don't think it matters when it's run inside WaitAssertion either way, but it's probably better to do server.ResolveDependency anyway.